src/components/Tabs/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Tab from "./Tab";
+
+const Tabs = ({ children }) => {
+  const [state, setState] = useState({ activeTab: children[0].props.label });
```

Comment:
I like the way you use the state to keep track of the active tab! As this state only keep tracks of the active tab, you can change its implementation to something like

```
const [activeTab, setActiveTab] = useState(children[0].props.label);
```

So it would be a bit easier to use and read